pennylane/pauli/pauli_arithmetic.py

```diff
@@ -80,6 +81,20 @@ def _cached_arange(n):
     return np.arange(n)
 
 
+pauli_to_sparse_int = {I: 0, X: 1, Y: 1, Z: 0}  # (I, Z) and (X, Y) have the same sparsity
+
+
+def _ps_to_sparse_index(pauli_words, wires):
+    """Represent the Pauli words sparse structure in a matrix of shape n_words x n_wires."""
+    indices = np.zeros((len(pauli_words), len(wires)))
+    for i, pw in enumerate(pauli_words):
+        if not pw.wires:
+            continue
+        wire_idx = np.array(wires.indices(pw.wires))
```

Comment:
`idx` to me indicates "single index".  Maybe `wire_indices` would be a bit better to indicate that its multiple indices?

Reply:
Yup, I agree!